examples/test_data_gen/src/construct_test_data_flow/flow.dag.yaml

```diff
@@ -0,0 +1,138 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
+environment:
+  python_requirements_txt: requirements.txt
+inputs:
+  text_chunk:
+    type: string
+    is_chat_input: false
+  text_meta:
+    type: string
+    default: "{}"
+outputs:
+  question:
+    type: string
+    reference: ${validate_test_question.output}
+  ground_truth:
+    type: string
+    reference: ${validate_ground_truth.output.ground_truth}
+  debug_info:
+    type: string
+    reference: ${generate_debug_info.output}
+nodes:
+- name: validate_text_trunk_prompt
+  type: prompt
+  source:
+    type: code
+    path: validate_text_trunk_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+  use_variants: false
+- name: validate_seed_question_prompt
+  type: prompt
+  source:
+    type: code
+    path: validate_question_prompt.jinja2
+  inputs:
+    question: ${validate_and_generate_seed_question.output.question}
+  use_variants: false
+- name: seed_question_prompt
+  type: prompt
+  source:
+    type: code
+    path: seed_question_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+  use_variants: false
+- name: generate_ground_truth_prompt
+  type: prompt
+  source:
+    type: code
+    path: generate_ground_truth_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+    question: ${validate_test_question.output}
+  use_variants: false
+- name: validate_and_generate_seed_question
+  type: python
+  source:
+    type: code
+    path: validate_and_generate_seed_question.py
+  inputs:
+    connection:
+    model: gpt-4
```

Comment:
please expose common model tuning parameters on this python tool like `temperature`, `max_tokens`, ....